### PR TITLE
100 quiz

### PR DESCRIPTION
**Pull request by jericoatos**

### CODE REVIEW

**Review by connmor2003 — APPROVED** [review]

**Review by orlaghmccloskey1 — COMMENTED** [review]
make continue green and back button grey - see below from UX designer 
image

**Review by orlaghmccloskey1 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -17,3 +17,7 @@
 
 1. Run command 'npm test'
 
+## How to Run Quiz
+
+1. Add OpenAI api key to .zshrc file
```
Comment (orlaghmccloskey1) [inline README.md]: mention user saving it as OPENAI_API_KEY in .zshrc file as thats whats being called in code

File: .github/workflows/docker.yml [hunk 2]
```diff
@@ -19,6 +19,7 @@ jobs:
         aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
         aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
         aws-region: eu-west-1
+        openai-api-key: ${{ secrets.OPENAI_API_KEY }}
```
Comment (mlynch147) [inline .github/workflows/docker.yml]: i dont think this is needed...

**Review by orlaghmccloskey1 — DISMISSED** [review]

File: .github/workflows/docker.yml (hunk 2 above)
Comment (jericoatos) [reply .github/workflows/docker.yml]: change added

File: README.md (hunk 1 above)
Comment (jericoatos) [reply README.md]: change added

**Review by orlaghmccloskey1 — APPROVED** [review]

**Pull request merged**